opt_probs.py

```diff
@@ -268,7 +268,7 @@ def eval_node_probs(self):
                 subset = self.keep_sample[np.where(
                     self.keep_sample[:, parent[i - 1]] == j)[0]]
 
-                if len(subset) == 0:
+                if not subset:
```

Comment:
As with the previous comment, how about we go with this:
```suggestion
                if not len(subset):
```